src/app/app.module.ts

```diff
@@ -107,6 +107,7 @@ import { FirestoreTimePipe } from './pipes-and-directives/firestore-time.pipe';
 import { DemoCardWrapperComponent } from './components/wrapper-components/demo-card-wrapper/demo-card-wrapper.component';
 import { GameEventService } from './services/GameEventService';
 import { HivePieceComponent } from './games/hive/hive-piece.component';
+import { DiaballikComponent } from './games/diaballik/diaballik.component';
```

Comment:
ERROR Error: Uncaught (in promise): Error: Assertion failure: We received an incorrect db move: DiaballikMove((1, 6) -> (1, 5), MGPOptional.of((1, 5) -> (1, 4)), MGPOptional.of((3, 6) -> (1, 4))) at turn 0because "Cannot pass without the ball"

Reply:
ah putain c'est une couille d'encodeur, il encode des coord to coord peu importe si c'est une passe ou pas :D

Reply:
y avais pas encore de disjunction ?

Reply:
ça a été fixé donc !